05/app.js

```diff
@@ -1,3 +1,44 @@
 console.log('DOM');
 
 const curr = document.querySelector('.js-curr');
+if (!curr) {
+    console.error('.js-curr elem not found');
+} else {
```

Comment:
Jakoś osobiście wolę zawsze wybierać ten "normalny flow" programu, a wyjątki obsługuję potem czyli zrobiłbym raczej `if(curr) {`, poza tym dodawanie wykrzyknika to dodatkowa, niepotrzebna operacja - faktycznie w tym przypadku nic nie znaczy, ale może będziesz miał przypadek, w którym ta różnica będzie istotna ;P